SConstruct

```diff
@@ -1487,10 +1485,10 @@ if testEnv["TEST_LIBPATH"] != "" :
 	testEnvLibPath = testEnv["TEST_LIBPATH"] + os.pathsep + testEnvLibPath
 testEnvLibPath = testEnv.subst( testEnvLibPath )
 
-testEnv["ENV"][testEnv["TEST_LIBRARY_PATH_ENV_VAR"]] = testEnvLibPath
-if libraryPathEnvVar :
-	testEnv["ENV"][libraryPathEnvVar] = testEnvLibPath
-testEnv["ENV"]["IECORE_OP_PATHS"] = "test/IECore/ops"
+if "TEST_LIBRARY_PATH_ENV_VAR" in testEnv and testEnv["TEST_LIBRARY_PATH_ENV_VAR"] :
```

Comment:
Maybe I'm missing something obvious, but I don't see how this can fail either. I get the `testEnv["ENV"].get(testEnv["TEST_LIBRARY_PATH_ENV_VAR"], "")` on the line below, but not this line.

Reply:
fixed in https://github.com/ImageEngine/cortex/pull/1181/commits/a641c41f73c4b9598b3fd81975c5f3cb5a1dadf8